Description: make package build reproducible
 By disabling the HTML timestamp setting in Doxygen.
Author: Ghislain Antony Vaillant <ghisvail@gmail.com>
Forwarded: not-needed
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/doc/clBLAS.doxy
+++ b/doc/clBLAS.doxy
@@ -974,7 +974,7 @@
 # page will contain the date and time when the page was generated. Setting 
 # this to NO can help when comparing the output of multiple runs.
 
-HTML_TIMESTAMP         = YES
+HTML_TIMESTAMP         = NO
 
 # If the HTML_DYNAMIC_SECTIONS tag is set to YES then the generated HTML 
 # documentation will contain sections that can be hidden and shown after the 
